package.json

```diff
@@ -83,5 +202,8 @@
   },
   "devDependencies": {
     "grunt": "~0.4.5"
+  },
+  "scripts": {
+    "prepublish": "node utils/updatePackage.js"
```

Comment:
Let's change this to a different phase since "prepublish" runs on `npm install` as well.